packages/components/src/tooltip/test/index.js

```diff
@@ -11,60 +11,55 @@ import Tooltip from '../';
  * WordPress dependencies
  */
 import { TOOLTIP_DELAY } from '../index.js';
-import { act } from '@testing-library/react';
 
 describe( 'Tooltip', () => {
 	describe( '#render()', () => {
 		it( 'should render children (abort) if multiple children passed', () => {
-			// Mount: Enzyme shallow does not support wrapping multiple nodes.
-			const wrapper = mount(
+			const { container } = render(
 				<Tooltip>
 					<div />
 					<div />
 				</Tooltip>
 			);
 
-			expect( wrapper.children() ).toHaveLength( 2 );
+			expect( container.children ).toHaveLength( 2 );
 		} );
 
 		it( 'should render children', () => {
-			const wrapper = shallow(
+			render(
 				<Tooltip position="bottom right" text="Help text">
 					<button>Hover Me!</button>
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-			expect( wrapper.type() ).toBe( 'button' );
-			expect( button.children() ).toHaveLength( 1 );
-			expect( button.childAt( 0 ).text() ).toBe( 'Hover Me!' );
+			const button = screen.getByRole( 'button' );
+			expect( button.nodeName ).toBe( 'BUTTON' );
+			expect( button.childNodes ).toHaveLength( 1 );
+			expect( button ).toHaveTextContent( 'Hover Me!' );
```

Comment:
These lines can be replaced by

```suggestion
			expect(
				screen.getByRole( 'button', { name: 'Hover Me!' } )
			).toBeInTheDocument();
```

This follows better the RTL style of testing:

- we don't need to know that the tagnane is a `BUTTON`, as that would be testing the implementation details. Instead, all we should care about is that, from the point of view of the user, what is rendered to screen is semantically a `button`
- Instead of checking the length of its `childNodes` and its `textContent`, we can just use the `name` option in the `getByRole` query

Reply:
Fixed by cdafb54 👍